build.gradle.kts

```diff
@@ -89,45 +53,13 @@ tasks {
             kotlinOptions.jvmTarget = "1.8"
         }
     }
-
-    withType<Detekt> {
-        jvmTarget = "1.8"
-    }
-
     patchPluginXml {
-        version(pluginVersion)
-        sinceBuild(pluginSinceBuild)
-        untilBuild(pluginUntilBuild)
-
-        // Extract the <!-- Plugin description --> section from README.md and provide for the plugin's manifest
-        pluginDescription(
-            closure {
-                File("./README.md").readText().lines().run {
-                    val start = "<!-- Plugin description -->"
-                    val end = "<!-- Plugin description end -->"
```

Comment:
This weird magic was parsing the README to get a description of the plugin. I got rid of to reduce build complexity & instead just added the plugin description permanently in the plugin.xml.